### PR TITLE
[DOC] data-ingestion-and-preparation README edits & new grafwiz tutorial doc review + edit tutorial titles

### PR DESCRIPTION
**Pull request by Sharon-iguazio**
@gilad-shaham please review all edits.

@arthurgarmider-igz / @zilbermanor  please review the changes to the  **grafana-grafwiz.ipynb** NB and see my related questions on Slack.
UPDATE: Or approved the changes and I made some updates after consulting Or and Adrian.

### CODE REVIEW

File: data-ingestion-and-preparation/v3io-objects.ipynb [hunk 1]
```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Using Iguazio's Data-Object API"
+    "# Using Paltform's Data-Object API"
```
Comment (gilad-shaham) [inline data-ingestion-and-preparation/v3io-objects.ipynb:7]: typo

File: data-ingestion-and-preparation/grafana-grafwiz.ipynb [hunk 2]
```diff
@@ -211,33 +415,42 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploying the Dashboard to Grafana\n",
+    "\n",
+    "Deploy the new Grafana dashboard to your Grafana service.\n",
+    "\n",
+    "> **Note:** Allow a few minutes for the dashboard loading to complete. "
```
Comment (gilad-shaham) [inline data-ingestion-and-preparation/grafana-grafwiz.ipynb:442]: Did it take you a few minutes?

**Review by gilad-shaham — CHANGES_REQUESTED** [review]

File: data-ingestion-and-preparation/v3io-objects.ipynb (hunk 1 above)
Comment (Sharon-iguazio) [reply data-ingestion-and-preparation/v3io-objects.ipynb]: FIXED (also added "the" before "Platform's") — commit 6611b5c.

File: data-ingestion-and-preparation/grafana-grafwiz.ipynb (hunk 2 above)
Comment (Sharon-iguazio) [reply data-ingestion-and-preparation/grafana-grafwiz.ipynb:442]: I didn't test it. I didn't add this text, I just edited and moved it from a previous cell, and I asked Arthur and Or to confirm the move on Slack (I'm still awaiting their reply; they're also reviewers in this PR). The is from the original NB:
```markdown
 #### Create a table and 2 graphs
 Please note that you have to wait for couple of minutes to see the graphs being updated
```

**Review by zilbermanor — APPROVED** [review]

**Pull request merged**